tests/unittests/lit.cfg.py

```diff
@@ -16,6 +16,9 @@
 
 config.environment['HALO_BUILD_DIR'] = config.halo_build_dir
 config.environment['HALO_SRC_DIR'] = config.halo_src_dir
```

Comment:
set environment variables repeatedly in run_list.sh line 4-6

Reply:
Right. I will remove them